devcycle.rst

```diff
@@ -292,6 +292,10 @@ Current Owners
 | Łukasz Langa         | CPython Developer in Residence | ambv            |
 +----------------------+--------------------------------+-----------------+
 
+``@python/organization-owners`` can be mentioned across the GitHub organization
+to request that one of the Organization Owners address an issue that requires
+the access levels granted soley to `permissions level <https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization#permissions-for-organization-roles>`_.
```

Comment:
```suggestion
Certain actions (blocking spam accounts, inviting new users, adjusting
organisation-level settings) can only `be performed`_ by owners of the Python
organisation on GitHub. The ``@python/organization-owners`` team can be
mentioned to request assistance from an organisation owner.

.. _be performed: https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization#permissions-for-organization-roles
```

A

Reply:
I like this!
I just have a minor note: the clickable text of a link (in this case `be performed`) should be somewhat representative of the content of the link.  Using `owners of the Python organization` might be better, since the link is about organization roles.

Reply:
I chose "be performed" as the link is a permissions link not a description of org owners -- I'm not too fussy though -- please steal & improve my words!

A

Reply:
For future reference: please use bastardi**z**ed English in the docs.